### PR TITLE
Attach lazy retick to getComputedStyle after inline style change

### PR DESCRIPTION
**Pull request by alancutter**
Setting the inline style of an animated element causes the animation frame to completely re-evaluate so that getComputedStyle() can be accurate with respect to the new inline style and any animations ontop of it.

Users that set hundreds of inline styles per frame without calling getComputedStyle are punished with a useless animation tick after each set. This change fixes that.

### CODE REVIEW

File: web-animations.js [hunk 1]
```diff
@@ -4819,6 +4819,16 @@ AnimatedCSSStyleDeclaration.prototype = {
   _setAnimatedProperty: function(property, value) {
     this._style[property] = value;
     this._isAnimatedProperty[property] = true;
+  },
+  _inlineStyleChanged: function() {
+    maybeRestartAnimation();
```
Comment (dstoc) [inline web-animations.js:4824]: Why is this needed? Perhaps it should have a return value to indicate whether a tick will happen or not?

Comment (alancutter) [inline web-animations.js:4824]: Without calling maybeRestartAnimation() the element won't have its animation reapplied on the next frame if no animations are running and getComputedStyle() is never called.
I'm not sure what a return value would be used for.

Comment (dstoc) [inline web-animations.js:4824]: I looked further, maybeRestartAnimation doesn't do what I thought it would do. I assumed it would only schedule at tick if one was required. So this seems OK.

Comment (alancutter) [inline web-animations.js:4824]: Perhaps it should be named ensureTickScheduled(). :)
